superset/config.py

```diff
@@ -1327,6 +1327,8 @@ def EMAIL_HEADER_MUTATOR(  # pylint: disable=invalid-name,unused-argument
 # Some sqlalchemy connection strings can open Superset to security risks.
 # Typically these should not be allowed.
 PREVENT_UNSAFE_DB_CONNECTIONS = True
+# Allows users to add a ``superset://`` DB that can query across databases.
+ENABLE_SUPERSET_META_DB = False
```

Comment:
I know we aren't very consistent in using configs vs feature flags, but should this be a FF until it's been considered stable?

Reply:
Ah, good point, this should be a feature flag! I'll move it over!

Reply:
Done!